src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  box-sizing: border-box;
```

Comment:
This rule is not inheriting, no need to set it to body (we didn't set any paddings or borders to body)